netpyne_ui/netpyne_geppetto.py

```diff
@@ -331,6 +331,16 @@ def getNetPyNELFPLocationsPlot(self):
             svgs.append(ui.getSVG(fig[0][0]))
         
         return svgs.__str__()
+
+    def getNetPyNERxDConcentration(self):
+        try:
+            fig = rxd.plotExtracellularConcentration(species=rxd.ca)
```

Comment:
Is it not needed to import rxd for this to work?

Reply:
this is just a temporary hack for the tutorial, so for this to work the user has to run from the jupyter nb: `import gui_tut3_rxd as rxd`

Reply:
Actually, you can just do 'from neuron import crxd as rxd', can't you? and we are already doing sth similar with matplotlib here https://github.com/MetaCell/NetPyNE-UI/blob/e00052075a5ce739381998fc799c943f0b01325f/netpyne_ui/netpyne_geppetto.py#L21

If what I am saying is correct is almost the same thing we are doing with matplotlib and I wouldn't consider it as hacky solution.

Reply:
it's not the same as rxd -- I shouldn't really have called it rxd (it was just to make it shorter).
the module I'm importing is custom made for the tutorial: https://github.com/Neurosim-lab/netpyne_workspace/blob/CNS18/gui_tut3_rxd.py